client/components/credit-card/stored-card.jsx

```diff
@@ -9,6 +9,27 @@ import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import { localize } from 'i18n-calypso';
 
+export const getCreditCardSummary = ( translate, type, digits ) => {
+	const supportedTypes = {
+		amex: translate( 'American Express' ),
+		diners: translate( 'Diners Club' ),
+		discover: translate( 'Discover' ),
+		jcb: translate( 'JCB' ),
+		mastercard: translate( 'Mastercard' ),
```

Comment:
Hi! I've found a possible matching string that has already been translated **52** times:
`translate( 'MasterCard' )` *ES Score: 32*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/enhance-credit-card-component-summary-and-responsiveness)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).

Reply:
Hm, looks like the brand's stylization was changed from "MasterCard" to "Mastercard" in 2016, but we don't necessarily want to lose the translations...

Reply:
cc @Automattic/i18n

Reply:
TIL the capitalization change.
Since this is a newly-added string, we won't lose current translations because of this. We would want to update all the existing strings with the deprecated capitalization in the near future, though :)